### ISSUE TITLE
Encoding rule for default clearance depth for Wreck

### ISSUE THREAD

**Issue by FrankHippmann** [iho-ohi/S-101-Documentation-and-FC#188]
The DCEG 13.5 states:

> The ECDIS "system" attribute **default clearance depth** must be populated with a value, which must not be an empty (null) value, if the attribute **height** has not been populated and the attribute **category of wreck** is populated or the attribute **value of sounding is populated** with an empty (null) value.

This implies that **default clearance depth** must be populated for the following Wreck, for example:

**category of wreck**:  1 : non-dangerous wreck

My thinking based on my understanding is as follows.

Can we express the condition more simply as:

> if the attribute **value of sounding** is populated with an empty (null) value.

The rationale is that if **value of sounding** is empty/unknown (null), then **height** and **category of wreck** must not be present. Similarly for **Obstruction** and **Underwater/awash rock**, if **value of sounding** is empty/unknown (null), then **height** must not be present.

To put it another way, **default clearance depth** is only required as a substitute for **value of sounding**, i.e., when **value of sounding** is unknown and encoded as null.

**Comment by FrankHippmann** [comment]
There was some confusion relating this issue at S-101PT 15. This issue is not about whether or not **default clearance depth** is required by S-101, rather it is about an apparent inconsistency in wordings of the DCEG. I will try to express this another way. The following are three statements taken from the DCEG.

1. The attribute **default clearance depth** must be populated with a value, which must not be an empty (null) value, only if the attribute **value of sounding** for the feature instance is populated with an empty (null) value and the attribute **height**, if an allowable attribute for the feature, is not populated.
2. _Exactly one of the attributes **category of wreck** or **value of sounding** must be populated._ That is, **value of sounding** must not be populated when **category of wreck** is populated.
3. ... **default clearance depth** must be populated with a value if the attribute **height** has not been populated and the attribute **category of wreck** is populated or the attribute **value of sounding** is populated with an empty (null) value.

These three statements contradict one another. According to statement 3, a wreck with **category of wreck** 1 (non-dangerous wreck) and no **height** attribute must have **default clearance depth** populated, and according to statement 2, **value of sounding** must not be populated. This contradicts statement 1.

Why would **default clearance depth** be required in a circumstance where **value of sounding** is prohibited, i.e., when **category of wreck** is populated?
